core/src/banking_stage.rs

```diff
@@ -928,14 +928,19 @@ impl Service for BankingStage {
 pub fn create_test_recorder(
     bank: &Arc<Bank>,
     blocktree: &Arc<Blocktree>,
+    poh_config: Option<PohConfig>,
 ) -> (
     Arc<AtomicBool>,
     Arc<Mutex<PohRecorder>>,
     PohService,
     Receiver<WorkingBankEntry>,
 ) {
     let exit = Arc::new(AtomicBool::new(false));
-    let poh_config = Arc::new(PohConfig::default());
+    let poh_config = if let Some(poh_config) = poh_config {
```

Comment:
Hmm, I'm sure there is better way to write in more idiomatic Rust way.

Reply:
Maybe `let poh_config = Arc::new(poh_config.unwrap_or(PohConfig::default()))` :D